### DISCUSSION TITLE
Allure doesn't count skipped tests in Cypress

### DISCUSSION THREAD

**Discussion by YuriyBusiness** [allure-framework/allure2#2584] [Questions & Support] [answered]
Hi, all.
I'm using Allure reporter, Cypress 12.8.3, and cypress-fail-fast plugin. cypress-fail-fast will stop test run on the first failure, I use this plugin to save time in the pipeline.
The problem. For example total tests: 10, passed: 5, failed:1(because cypress-fail-fast stops test run on first failure ) and skipped:4. So allure will show a total: 6, passed:5, failed:1. this behavior is understandable.
Question: maybe someone knows, how to say Allure "count all tests" to show like total tests: 10, passed: 5, failed:1 and skipped:4.

**Comment by baev** [comment]
The issue https://github.com/allure-framework/allure-js/issues/948
The pull request https://github.com/allure-framework/allure-js/pull/949

**Reply by epszaw** [reply]
The feature was implemented [here](https://github.com/allure-framework/allure-js/pull/1029) and will be available in the next release.

**Reply by YuriyBusiness** [reply]
> The feature was implemented [here](https://github.com/allure-framework/allure-js/pull/1029) and will be available in the next release.

awesome thanks!

**Comment by baev** [comment, marked as answer]
fixed in `allure-cypress@3.0.0-beta.5`
